src/QuadVelocityController.cpp

```diff
@@ -122,19 +128,46 @@ bool QuadVelocityController::getVelocities(geometry_msgs::Vector3& return_veloci
                 velocities_valid = false;
             }
 
-            // Get the transforms with the stamps for readability
+            // Get the transforms without the stamps for readability
             geometry_msgs::Transform& transform = transformStamped.transform;
             geometry_msgs::Transform& oldTransform = last_transform_stamped_.transform;
 
+            // Get the yaw (z axis) rotation from the quanternion
+            double ysqr = transform.rotation.y * transform.rotation.y;
```

Comment:
This is only running at like 100Hz, if the jetson can't afford to create 200 extra objects per second we have other problems